### PR TITLE
Hw03

### PR DESCRIPTION
**Pull request by eburkey**
@sejalkesh @tylerallan

### CODE REVIEW

File: HW03/hw03.Rmd [hunk 1]
```diff
@@ -61,45 +66,40 @@ reversed <- dictionary %>%
...
 ```{r}
-green_data %>%
-  rowwise() %>%
-  group_by(id) %>%
-  mutate(total_green = sum(C(green1 : green5), na.rm = TRUE),
-          total_comp = sum(C(comp1 : comp10), na.rm = TRUE),
-          total_intel = sum(C(intel1 : intel10),na.rm = TRUE),
-          total_open = sum(C(open1 : open10),na.rm = TRUE))%>%
+Green_R <- green_data %>%
+  mutate(across(all_of(reversed),
```
Comment (tylerallan) [inline HW03/hw03.Rmd:84]: I'm not quite seeing where you pulled out the items that needed to be reversed and put them into the object "reversed." I could just be missing it here, but you only need to reverse certain variables.

File: HW03/hw03.Rmd [hunk 2]
```diff
@@ -108,11 +108,15 @@ Rescale the variables so that they go from 0-100 instead of the original range.
      Name the recaled variables `*_pomp`.
 
 ```{r}
-DataSet3 <- green_data %>%
-  mutate(across(all_of(reversed), 
-                ~ recode(.x, "1" = 0, "2" = 25, "4" = 75, "5" = 100))) %>%
-  
-  
+Green_Rescale <- green_data %>%
+  mutate(across(all_of(reversed),~ recode(.x, "1" = 0, "2" = 25, "4" = 75, "5" = 100))) %>%
```
Comment (tylerallan) [inline HW03/hw03.Rmd:112]: You'll want to add "3" = 50 in your final version. Also you want to mutate across all of your original dataset not just the variables you pulled out to be reversed.

**Comment by tylerallan** [comment]
Peer Review of HW-03 for E Burkey
----------------------------------------------------
Topic                       | Excellent | Satisfactory | Needs Work |
----------------------------|-----|-----|-----|
**Coding style**            |  | :heavy_check_mark:   |  |
**Coding strategy**         |  | :heavy_check_mark:  |  |
**Presentation: graphs**    |  |  | :heavy_check_mark: |
**Presentation: tables**    | :heavy_check_mark:   |   |  |
**Achievement, creativity** | :heavy_check_mark: |  |  |
**Ease of access**          |  |  | :heavy_check_mark:  |

Remarks:

- Your commits are a little bare bones. Maybe consider adding more info so you know what each commit contained rather than just saying updates. 
- Great job working through the homework! Your coding is structured well.
- I think you may have gotten confused concerning the "reversed" part of the mutate code. That was a specific set of data that we wanted to reverse. For that section of code you'll want to include the data.

File: HW03/hw03.Rmd [hunk 3]
```diff
@@ -61,45 +66,40 @@ reversed <- dictionary %>%
...
+                         "2" = 4,
+                         "1" = 5)))
 ```
 
-```{r}
+- Recoded the data to organize it properly and categorize it in a way that is more clear 
+
 
+```{r}
+Green_Calculations <- green_data %>%
```
Comment (sejalkesh) [inline HW03/hw03.Rmd:97]: Good job with keeping your code organized and easy to understand!

File: HW03/hw03.Rmd [hunk 4]
```diff
@@ -61,45 +66,40 @@ reversed <- dictionary %>%
...
-                         "3" = "3",
-                         "2" = "4",
-                         "1" = "5"), 
-                        .names = "{.col}r"))
-```
-
-- Recoded the data to organize it properly and categorize it in a way that is more clear 
-
 ```{r}
 green_data %>%
```
Comment (sejalkesh) [inline HW03/hw03.Rmd:75]: You did a great job here!

**Pull request merged**